spec/models/api_request_spec.rb

```diff
@@ -10,8 +10,9 @@
     end
 
     describe "with valid host, path, and configure_auth" do
+      let(:api) { ApiRequest.new }
```

Comment:
[huh, non-blocking] I thought I tried this when I wrote this test, and finally decided to stick it in as a `@` instead when it wouldn't work. Maybe I was trying to use `let` in the before block and am misremembering!  HMMM.  Well, good to know it works.